### PR TITLE
separated  route from server.js

### PR DESCRIPTION
**Pull request by wastech**

### CODE REVIEW

File: route.js [hunk 1]
```diff
@@ -0,0 +1,65 @@
+const express = require("express");
```
Comment (impactmass) [inline route.js:1]: Prefer not to mix `require` with `import`. Let's stick with one: `import`
```suggestion
import express from "express";
```

File: route.js [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+
+  return res.redirect(
+    `/#${querystring.stringify({
+      access_token: data.access_token,
+      refresh_token: data.refresh_token,
+    })}`
+  );
+});
+
+module.exports = route;
```
Comment (impactmass) [inline route.js:65]: Use export. So that we are sticking with import / export ES7 style https://javascript.info/import-export

File: server.js [hunk 3]
```diff
@@ -1,73 +1,16 @@
-import axios from "axios";
 import config from "./config";
 import cookieParser from "cookie-parser";
 import express from "express";
-import { generateRandomString } from "./utils";
 import morganLogger from "morgan";
-import querystring from "querystring";
+const router = require("./route");
```
Comment (impactmass) [inline server.js:5]: import

File: server.js [hunk 4]
```diff
@@ -1,73 +1,16 @@
...
-} = config;
+const { PORT: port } = config;
 
 const app = express();
 
 app
   .use(express.static(`${__dirname}/public`))
   .use(morganLogger("dev"))
   .use(cookieParser())
+  .use("/auth", router)
```
Comment (impactmass) [inline server.js:15]: 💯 ❤️

File: public/index.html [hunk 5]
```diff
@@ -19,7 +19,9 @@ <h1 class="display-4">tiny</h1>
         <hr class="my-4" />
       </div>
       <div id="login">
-        <a class="btn btn-primary btn-lg" href="/login">Log in with Spotify</a>
+        <a class="btn btn-primary btn-lg" href="/auth/login"
+          >Log in with Spotify</a
```
Comment (impactmass) [inline public/index.html:23]: Let's not put line breaks in weird sports

File: route.js [hunk 6]
```diff
@@ -0,0 +1,65 @@
+const express = require("express");
+import axios from "axios";
+const route = express.Router();
```
Comment (impactmass) [inline route.js:3]: Put all your imports together. Then put all your declarations together. It helps code readability.

```
import a from b;
import c from d;
import e from f;

const abc = a()
const bcd = c()
```

File: public/index.html (hunk 5 above)
Comment (wastech) [reply public/index.html:23]: help me to check the setup again because it doesn't resolve

**Pull request closed without merging**